avo-hq/avo
# Only remove attachment upload button after trix has initialized

# Description


I'm finding that when attachments are disabled the trix field controller is trying to remove the attachments button before trix has initialized and before it has created the toolbar. This crashes the controller inside the connect method and so none of the other event listeners are set up and the attach files button isn't removed.


***NOTE:*** I haven't been able to test this myself, the code is my best guess at how to fix the issue.

Fixes # (issue)



# Checklist:



## Screenshots & recording




![Screenshot 2024-11-26 at 14 41 39](https://github.com/user-attachments/assets/e6f46160-33c4-4270-a9cb-fec98e4b7af6)

## Manual review steps


1. Find a resource with a trix field
2. Make sure it shows an attachment button
3. Disable attachments
4. Make sure the attachment button is now hidden

Manual reviewer: please leave a comment with output from the test if that's the case.
